benchmarks/benchmark_morphology.py

```diff
@@ -42,8 +41,14 @@ def peakmem_watershed(self, seed_count, connectivity, compactness):
 class Skeletonize3d(object):
 
     def setup(self, *args):
+        try:
+            # skip tests unless skeletonize supports 3d inputs
+            from skimage.morphology import skeletonize
+            skeletonize(np.ones((5, 5, 5)))
+        except (ImportError, ValueError):
+            raise NotImplementedError("3D skeltonize unavailable")
```

Comment:
This one is a bit more complicated. We originally had skeletonize (2D) and skeletonize3d. I then merged the APIs. So if skeletonize of a 3D volume fails, we should check for skeletonize3d, and if *that* fails, we should raise NotImplementedError.